src/components/pages/creative/controller/components/ImageFilters.js

```diff
@@ -17,6 +17,13 @@ const ImageFilters = () => {
         {filterObject: new fabric.Image.filters.Invert(), filterName: 'Invert'}
     ]
 
+
+    const c = function(num) {
+        return function(num2) {
+            return num + num2
+        }
+    }
```

Comment:
**Issue**: The function 'c' defined is a curried function that adds two numbers. However, it is not used anywhere within the code, leading to dead code which can confuse developers and lead to maintenance issues. <br> **Fix**: Remove the unused function to clean up the codebase. <br> **Code Suggestion**: 
 ```
 @@ -21,5 +21,0 @@
 -    const c = function(num) {
 -        return function(num2) {
 -            return num + num2
 -        }
 -    }
 ```